doc/fluid/design/dist_train/distributed_training_overview.md

```diff
@@ -0,0 +1,57 @@
+## distributed training overview doc
+
+Currently Paddle Fluid use parameter server architecture to support distributed training.
+
+For synchronous and asynchronous training, the differences are mostly in the logic of parameter server. Now we have already support synchronous training.
+
+### Synchronous training
+
+The training process of synchronous training is:
+
+![lookup table training](./src/sync_distributed_training.png)
```

Comment:
Better to use a "cross functional flowchart" to represent both procedure and communications.